runtime/hashing/build.gradle.kts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+description = "Hashing functions for Smithy services generated by smithy-kotlin"
+extra["displayName"] = "Smithy :: Kotlin :: Hashing"
+extra["moduleName"] = "aws.smithy.kotlin.runtime.hashing"
+
+kotlin {
+    sourceSets {
+        commonMain {
+            dependencies {
+                api(project(":runtime:utils"))
```

Comment:
nit add a comment about what we are exposing

Reply:
Actually, I've realized this can be `implementation` instead of `api`. Changing...